galsim/phase_screens.py

```diff
@@ -41,7 +41,7 @@ class AtmosphericScreen(object):
     @param altitude      Altitude of phase screen in km.  This is with respect to the telescope, not
                          sea-level.  [default: 0.0]
     @param time_step     Interval to use when advancing the screen in time in seconds.
-                         [default: 0.03]
+                         [default: 0.025]
```

Comment:
No, there wasn't a lot of thought behind this change.  I just wanted a time step that evenly divided 1.0 second.  I'm happy to set this back to 0.03 if you prefer.

I agree that the right thing to do would involve comparing to the asymptotic results, but unfortunately I think the convergence is likely to depend on a number of additional factors like the simulated wind speeds, exposure time, number of layers, telescope aperture, r0, ...  It might be useful to try some typical values for DES, LSST, HSC though.